anvio/terminal.py

```diff
@@ -351,6 +384,17 @@ def info(self, key, value, quiet=False, display_only=False, overwrite_verbose=Fa
         info_line = "%s%s %s: %s\n%s" % ('\n' * nl_before, c(label, lc),
                                          '.' * (self.width - len(label)),
                                          c(str(value), mc), '\n' * nl_after)
+        if align_long_values:
+            terminal_width = os.get_terminal_size()[0]
```

Comment:
there is already a global `get_terminal_size` function in `terminal.py` :)

I think it would be better if we stick with a single source of information for this.

Reply:
oops, I missed that :) this is now fixed.